python/lsst/ip/diffim/getTemplate.py

```diff
@@ -135,6 +140,19 @@ def run(self, exposure, sensorRef, templateIdList=None):
         return pipeBase.Struct(exposure=coaddExposure,
                                sources=None)
 
+    def getCoaddDatasetName(self):
+        """Return coadd name for given task config
+
+        Returns
+        -------
+        CoaddDatasetName : `string`
+
+        TODO: This nearly duplicates a method in CoaddBaseTask (DM-11985)
+        """
+        warpType = self.config.warpType
+        suffix = "" if warpType == "direct" else warpType[0].upper() + warpType[1:]
```

Comment:
This is probably the most efficient way of doing this, but using `warpType .title()` would be more clear.

Reply:
Oops. Yes, I didn't realize that would un-capitalize letters in the rest of the word. Good catch.